tests/unit_tests/Test/PM/Unit/ParseEnvVarJSON.cls

```diff
@@ -0,0 +1,87 @@
+Class Test.PM.Unit.ParseEnvVarJSON Extends %UnitTest.TestCase
+{
+
+Parameter ENVNAME = "mySecretEnvVarName";
+
+Parameter ENVVALUE = "This is a secret value";
+
+XData SampleJSON [ MimeType = application/json ]
+{
+{
+    "package": {
+        "nested": {
```

Comment:
Worthwhile adding examples with env vars at top level node, nested node, in an array and perhaps multiple env vars in a single value